### ISSUE TITLE
Is HoloLens support coming anytime soon?

### ISSUE THREAD

**Issue by ArmsOfSorrow** [MonoGame/MonoGame#5731]
It would be interesting to have support for "immersive" HoloLens apps as a lightweight alternative to Unity. Is this planned for the short term or are you waiting for a consumer device before supporting it?

**Comment by KonajuGames** [comment]
There hasn't been much movement on it. There is the SDK and emulator available, but hardware is still exceedingly expensive. $5K here in Australia. The current audience for HoloLens is also still very small.

**Comment by vpenades** [comment]
Hi, I just wanted to add a comment here: Since the last "[Creators Update](https://blogs.windows.com/buildingapps/2017/02/08/windows-developer-day-creators-update/)" recently released by Microsoft, the Hololens SDK, now rebranded as mixed reality SDK, is no longer tied exclusively to the HoloLens device, and is now open to [more headsets](https://arstechnica.com/gaming/2017/01/microsoft-to-have-holographic-developer-kits-for-cheap-vr-headsets-at-gdc/), other than the Hololens.

In other words, Microsoft has finally decided to define a common API for all VR devices, at least on the windows platform. Most probably, Oculus and Vive will not jump in immediately, but if other vendors begin introducing cheaper headsets, I think it's a matter of time.

**Comment by MaxxDelusional** [comment]
I talked to a Microsoft employee at BUILD about this. They would love for Monogame to support Windows Mixed Reality.

**Comment by demonixis** [comment]
@MaxxDelusional any news about this?

**Comment by theZMan** [comment]
I think Chad Carter might have been diong some HoloLens stuff... Calling @kewlniss

**Comment by DDReaper** [comment]
Been working towards this with a few MG devs, baby steps as it's different enough from other VR/XR platforms to need some thought.
See #5915 for more details.

**Comment by demonixis** [comment]
@theZMan I compiled some work on this [right here](https://github.com/demonixis/MGMixedReality), if you have an Immersive headset or a HoloLens, you can help us ;)

I've also contacted Microsoft and few other devs, I'm waiting... There is no real documentation about that or I didn't find it.

If anyone have any informations about the Holographic SDK it can be nice to share them. The D3D sample is just hard to read, but my work is based on it.
